django-backend/fecfiler/cash_on_hand/migrations/0002_alter_cashonhandyearly_cash_on_hand_and_more.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 5.1.1 on 2024-11-16 21:51
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cash_on_hand', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='cashonhandyearly',
+            name='cash_on_hand',
+            field=models.DecimalField(decimal_places=2, default=0, max_digits=11),
+            preserve_default=False,
+        ),
+        migrations.AlterField(
+            model_name='cashonhandyearly',
+            name='year',
+            field=models.TextField(default=2020, unique=True),
```

Comment:
I think we need to remove the unique as well.  Two committees can't override the same year otherwise.

Reply:
Thanks @toddlees! Change made in https://github.com/fecgov/fecfile-web-api/pull/1211/commits/98babe80ce3ab904785c49f99def1ffc088756d0 and test added in https://github.com/fecgov/fecfile-web-api/pull/1211/commits/079c0d4515db625ae6a1dcc9256f8024e1a4c92f. Please let me know if you want me to make any other changes.

Reply:
I think you can get rid of the unique in 0002 and no longer need 0003 as it's own migration

Reply:
Change made in https://github.com/fecgov/fecfile-web-api/pull/1211/commits/12392df95bf84003aad4bccb47d0c8c79f903351